testsuite/smoke-test/src/aptos/generic_publish.rs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Aptos
```

Comment:
I don't see why these files are on smoke-test folder?

They are there to regenerate compiled versions? maybe they should go to into something like either:
- crates/transaction-emitter-lib/src/module_publish/generate_compiled_module/
- testsuite/module_publish/src/generate_compiled_module/
what do you think?

Reply:
I had them there just because it was the simplest place to have them, as the entire infra to compile was there.
Happy to move to a better place, and I will try the transaction-emitter-lib

Reply:
added the module generator in `testsuites/module-publish`.
See if you like it